.github/workflows/mend_scan.yaml

```diff
@@ -0,0 +1,37 @@
+name: Mend Security Scan
+
+on:
+  schedule:
+    - cron:  '5 0 * * 0'
+
+jobs:
+  mend-scan:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code  
+      uses: actions/checkout@v4
+
+    - name: Set up Java 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Setup Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.19.x
+
+    - name: Download Mend Universal Agent
+      run: curl https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar -o ./wss-unified-agent.jar
+
+    - name: Run Mend Scan 
+      run: java -jar ./wss-unified-agent.jar -c $CONFIG_FILE -wss.url $WSS_URL -apiKey $API_KEY -userKey $USER_KEY -productToken $PRODUCT_TOKEN
+      env:
+        USER_KEY: ${{ secrets.MEND_USER_KEY }}
+        PRODUCT_TOKEN: ${{ secrets.MEND_SHC_PRODUCT_TOKEN }}
+        WSS_URL: ${{ secrets.MEND_URL }}
+        API_KEY: ${{ secrets.MEND_API_TOKEN }}
+        CONFIG_FILE: './.github/workflows/mend.config'
```

Comment:
I assume these exist this time? :D

Reply:
yes, they are maintained at GitHub org level :-)